easybuild/easyconfigs/m/matplotlib/matplotlib-2.1.0-foss-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'Bundle'
```

Comment:
@vanzod Why not use `PythonBundle` here too?

Reply:
For consistency with all other `matplotlib` easyconfig files already in the repo. 
I believe it might be cleaner to fix them in a separate PR than here. Thoughts?

Reply:
Well either we do it for new easyconfigs going forward, or we update all existing `matplotlib` easyconfigs to use `PythonBundle`. If you're up for the latter, I'm not complaining ;)